lib/puppet/functions/simplib/deprecation.rb

```diff
@@ -1,8 +1,8 @@
 # Function to print deprecation warnings, logging a warning once
 # for a given key.
 #
-# Messages can be suppressed if the SIMPLIB_LOG_DEPRECATIONS
-# environment is set to 'false'
+# Messages can be enabled if the SIMPLIB_LOG_DEPRECATIONS
```

Comment:
Is this going to be switched back in the future?

Reply:
@jeefberkey The intent is to switch it back when we are ready to fix the use of deprecated functions in the SIMP modules.  Right now, we are not done converting Puppet 3 functions to Puppet 4 and have, for the most part, have not converted modules using the old functions to use the new functions.

Reply:
SIMP-4071